sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsub/PubsubUnboundedSink.java

```diff
@@ -236,8 +237,8 @@ public void populateDisplayData(Builder builder) {
     private transient PubsubClient pubsubClient;
 
     private final Counter batchCounter = Metrics.counter(WriterFn.class, "batches");
-    private final Counter elementCounter = Metrics.counter(WriterFn.class, "elements");
-    private final Counter byteCounter = Metrics.counter(WriterFn.class, "bytes");
+    private final Counter elementCounter = IOMetrics.elementsWritten();
```

Comment:
The metrics are in the `io` namespace and the step in which you write to PubSub. Is that indicative enough?